common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -201,48 +195,61 @@ def test_fields_on_others_public_profile(self):
         Then I shouldn't see the profile visibility selector dropdown.
         Also `location`, `language` and `about me` fields are not editable.
         """
-        self.visit_other_profile_page(self.OTHER_USER, privacy=self.PRIVACY_PUBLIC)
-        self.visit_other_profile_page(self.MY_USER)
+        other_username, other_user_id = self._initialize_other_user(privacy=self.PRIVACY_PUBLIC)
+        self.log_in_as_unique_user()
+        profile_page = self.visit_profile_page(other_username)
+        profile_page.wait_for_public_fields()
+        self.assertFalse(profile_page.privacy_field_visible)
+        fields_to_check = self.PUBLIC_PROFILE_FIELDS
```

Comment:
This isn't my code, but the GitHub diff makes it look like I introduced it. I've removed the variable.